src/components/AdjMatrix/AdjMatrixMethods.ts

```diff
@@ -49,6 +49,7 @@ export class View {
   private highlightedNodes: { [key: string]: any[] } = {};
   private columnSelectedNodes: any[] = [];
   private mouseOverEvents: any;
+  private maxVal: number = -Infinity;
```

Comment:
```suggestion
  private maxNumConnections: number = -Infinity;
```
More descriptive name per Jack's suggestion